config/envs/rhpam-authoring-ha.yaml

```diff
@@ -30,10 +30,18 @@ console:
                     containerPort: 8001
                     protocol: TCP
                 env:
-                  - name: APPFORMER_ELASTIC_HOST
-                    value: "[[.ApplicationName]]-[[.Constants.Product]]index"
+                  - name: OPENSHIFT_DNS_PING_SERVICE_NAME
+                    value: "[[.ApplicationName]]-[[.Console.Name]]-ping"
+                  - name: OPENSHIFT_DNS_PING_SERVICE_PORT
+                    value: "8888"
```

Comment:
Please look at the common.yaml and avoid overwriting anything in these files, unless you are changing the values.